### ISSUE TITLE
layerClient.newConversationWithParticipants crashes app

### ISSUE THREAD

**Issue by Mortenest** [layerhq/Atlas-iOS#1141]
Hi, 
I'm having problems with layer since I both upgraded Atlas to 1.0.20, and Xcode to 7.3 (and swift to 2.2).

It seems that layer is not really made for conversations without using the ATLConversationListViewController. However I need to access my conversations arbitrarily from such a list, which is why I'm utilising newConversationWithParticipants.
My code:

```
func conversationWithParticipants(userIds:Set<String>) -> LYRConversation? {
        let layerClient = (UIApplication.sharedApplication().delegate as! AppDelegate).layerClient   
       var conversation:LYRConversation?
        do {
            conversation = try layerClient.newConversationWithParticipants(userIds, options: nil)
        } catch let error as NSError {
            if let conversation:LYRConversation = error.userInfo["Existing Conversation"] as? LYRConversation {
                return conversation
            }
            return nil
        }
        if conversation != nil {
            return conversation
        }
        return nil
    }
```

This has been working with previous versions of layer/swift at least, meaning that if a conversation does exist layerClient.newConversationWithParticipants will throw an exception, and I can pick up the conversation from that exception.

The first time I try this, I get a conversation out as expected.
Now when I try this for the second time, the app crashes on the line

`            conversation = try layerClient.newConversationWithParticipants(userIds, options: nil)
`

It actually crashes in a do/try.
Xcode breaks on "swift_bridgeNonVerbatimFromObjectiveC" and it looks like there is a problem with "value type is not bridged to Objective-C".

I'm not completely sure what this means, but it seems that there are some problems issues between the bridging of this function to swift.

Anyone who had similar experiences, or good ideas what to do about this?

**Comment by JRam13** [comment]
Did you update to Layer 0.20.0?

**Comment by Mortenest** [comment]
I'm using 1.0.20 which is the newest version afaik.

**Comment by Mortenest** [comment]
I am using Cocoapods, and I'm using LayerKit v. 0.20.1if that's what you mean?

**Comment by Mortenest** [comment]
I have once again reverted back to Atlas 1.0.14 and LayerKit 0.17.7 to make it work. That is the only stable combination I have found.

**Comment by kmahal** [comment]
@Mortenest we haven't been able to reproduce this in our SDK, or in a bridged swift header.  I'll look further into this and report back.

**Comment by JRam13** [comment]
I thought this was fixed in 0.20.2 / 1.0.22. @Mortenest, check again.

**Comment by kcoleman731** [comment]
@Mortenest any updates here? Otherwise we will close.
